testing/e2e/e2e_staking_test.go

```diff
@@ -79,23 +76,12 @@ func (s *BeaconKitE2ESuite) TestDepositRobustness() {
 	// s.Require().NoError(err)
 
 	// Bind the deposit contract.
-	dc, err := deposit.NewBeaconDepositContract(
+	dc, err := deposit.NewDepositContract(
 		gethcommon.HexToAddress(spec.DefaultDepositContractAddress),
 		s.JSONRPCBalancer(),
 	)
 	s.Require().NoError(err)
 
-	tx, err := dc.AllowDeposit(&bind.TransactOpts{
-		From:   genesisAccount.Address(),
-		Signer: genesisAccount.SignerFunc(chainID),
-	}, sender.Address(), NumDepositsLoad)
-	s.Require().NoError(err)
-
-	// Wait for the transaction to be mined.
-	receipt, err := bind.WaitMined(s.Ctx(), s.JSONRPCBalancer(), tx)
-	s.Require().NoError(err)
-	s.Require().Equal(coretypes.ReceiptStatusSuccessful, receipt.Status)
```

Comment:
q: why was this dropped?

Reply:
Part of the old deposit contract that was permissioned. No longer around